docs/guide/jsonrpc_protocol.md

```diff
@@ -536,6 +536,11 @@ Get a connection string for the provided connection.
         /// Indicates whether the password should be return in the connection string
         /// </summary>
         public bool IncludePassword { get; set; }
+
+        /// <summary>
+        /// Indicates whether the application name should be return in the connection string. Default is false.
```

Comment:
`Default is true`? False isn't the correct default...

Reply:
Yeah mean to put this for IncludePassword.